lib/publify/api.rb

```diff
@@ -0,0 +1,18 @@
+require 'net/http'
+
+module Publify
+  class API
+
+    def self.latest_links(limit)
+      begin
+        uri = URI("http://#{PUBLIFY_HOSTNAME}/articles.json")
+        response = Net::HTTP.get uri
+        JSON.parse(response).first(limit)
+      rescue Exception => e
+        Rails.logger.error(e.message)
+        []
+      end
+    end
+
+  end
+end
```

Comment:
Looks nice yet seems like wrapping it up in an object that, at the minute, we don't need.